src/NetMQ/Core/Utils/Clock.cs

```diff
@@ -67,6 +70,7 @@ static Clock()
                 s_rdtscSupported = false;
             }
 #endif
+            s_spinLock = new SpinLock();
```

Comment:
```suggestion
            s_spinLock = new SpinLock(enableThreadOwnerTracking: false);
```

I don't think we need thread ownership tracking ([see](https://docs.microsoft.com/en-us/dotnet/api/system.threading.spinlock.-ctor?view=netframework-4.8#remarks)).